src/__tests__/utils/arithmetic-helper.test.ts

```diff
@@ -0,0 +1,490 @@
+        const result = isValidArithmeticExpression(parameter);
+
+        expect(result).toBe(false);
+      });
+    
+      it('should return true for a valid expression with mixed numbers and variables.', () => {
+        const parameter = '=5+param1/3-2';
+        const result = isValidArithmeticExpression(parameter);
+
+        expect(result).toBe(true);
+      });
+    
+      it('should return true for a valid arithmetic expression with decimal numbers.', () => {
+        const parameter = '=5.5*3.2*param1';
+        const result = isValidArithmeticExpression(parameter);
+        expect(result).toBe(true);
+      });
+    });
+
+    describe('evaluateSimpleArithmeticExpression(): ', () => {
+      it('should evaluate a valid addition expression.', () => {
+        const parameter = '=2+3';
+        const result = evaluateSimpleArithmeticExpression(parameter);
+        
+        expect(result).toBe(5);
+      });
+    
+      // it('should evaluate a valid subtraction expression.', () => {
+      //   const parameter = '=10-4';
+      //   const result = evaluateSimpleArithmeticExpression(parameter);
+
+      //   expect(result).toBe(6);
+      // });
+    
+      it('should evaluate a valid multiplication expression.', () => {
+        const parameter = '=2*3';
+        const result = evaluateSimpleArithmeticExpression(parameter);
+
+        expect(result).toBe(6);
+      });
+    
+      it('should evaluate a valid division expression', () => {
+        const parameter = '=10/2';
+        const result = evaluateSimpleArithmeticExpression(parameter);
+
+        expect(result).toBe(5);
+      });
+    
+      it('should return the original parameter if there is no "=" sign', () => {
+        const parameter = '2+3';
+        const result = evaluateSimpleArithmeticExpression(parameter);
+
+        expect(result).toBe(5);
+      });
+    
+      it('should return the original parameter if the expression format is invalid', () => {
+        const parameter = '=2+3*5';
+        const result = evaluateSimpleArithmeticExpression(parameter);
+        expect(result).toBe('=2+3*5');
+      });
+    
+      it('should return the original parameter if it contains unsupported characters', () => {
+        const parameter = '=a+b';
+        const result = evaluateSimpleArithmeticExpression(parameter);
+        expect(result).toBe('=a+b');
+      });
+    
+      it('should return the original parameter if it is not a simple expression', () => {
+        const parameter = '=param1*2';
+        const result = evaluateSimpleArithmeticExpression(parameter);
+        expect(result).toBe('=param1*2');
+      });
+    
+      it('should return the original parameter if it is a standalone number', () => {
+        const parameter = '=42';
+        const result = evaluateSimpleArithmeticExpression(parameter);
+        expect(result).toBe('=42');
+      });
+    
+      it('should return the original parameter if the expression contains decimals', () => {
```

Comment:
thanks for pointing out this case, I've fixed and now the result should be 6. Please update test description